server/templates/static/homepage.html

```diff
@@ -84,7 +84,7 @@ <h2>Explore the data</h2>
           Explore the data using one of our visualization tools (<a href="{{ url_for('tools.scatter') }}">Scatter</a>, <a href="{{ url_for('tools.map') }}">Map</a>,
           <a href="{{ url_for('tools.timeline') }}">Timeline</a>) or use the <a href="https://docs.datacommons.org/api">APIs</a> via Python notebooks or build your own
           application. Explore the thousands of variables in the <a href="{{ url_for('tools.stat_var') }}">Data Dictionary
-          Exploration Tool</a>. Contribute to the project on <a target="blank" href="https://github.com/datacommonsorg">Github {{ external_icon() }}</a>.
+          Exploration Tool</a>. Contribute to the project <a target="blank" href="https://docs.datacommons.org/contributing">here</a>.
```

Comment:
Some suggestions for wording (instead of just linking "here"):

[Contribute to the project](#).
Learn how you can [contribute to the project](#).

Reply:
done